src/serial/SerialInputHandler.cpp

```diff
@@ -220,6 +220,28 @@ void _handleAuthtokenCommand(char* arg, std::size_t argLength) {
   }
 }
 
+void _handleLcgOverrideCommand(char* arg, std::size_t argLength) {
+  if (arg == nullptr || argLength == 0) {
+    std::string lcgOverride;
+    if (!Config::GetBackendLCGOverride(lcgOverride)) {
+      SERPR_ERROR("Failed to get LCG override from config");
+      return;
+    }
+
+    // Get LCG override
+    SERPR_RESPONSE("LcgOverride|%s", lcgOverride.c_str());
+    return;
+  }
+
+  bool result = OpenShock::Config::SetBackendLCGOverride(std::string(arg, argLength));
+
+  if (result) {
+    SERPR_SUCCESS("Saved config");
```

Comment:
Should we do the same domain verification we did for the `domain` command?

Having an incorrect entry won't cause as erratic behavior as with an invalid `domain`, but maybe just for consistency/clearing?